src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -216,8 +224,9 @@
     <Exec Command="chmod u+x $(_crossGenPath)"
           Condition="'$(OSGroup)' != 'Windows_NT'" />
     <ItemGroup>
+      <!-- -->
```

Comment:
Empty comment?

Reply:
I will add comment to explain why S.R.WR in non-windows is excluded from crossgen..